HW_01/decorator_for_measuring_memory.py

```diff
@@ -0,0 +1,48 @@
+import functools
+import tracemalloc
+import requests
+
+
+def used_memory_deco(func):
+    """
+    A decorator to measure the memory usage of a function using tracemalloc
+    """
+
+    @functools.wraps(func)
+    def wrapper(*args, **kwargs):
+        tracemalloc.start()
+        # function call
+        result = func(*args, **kwargs)
+        # store displaying the memory
+        list_memory_data = tracemalloc.get_traced_memory()
```

Comment:
here is better to use unpack:
memory_one, memory_two = tracemalloc.get_traced_memory()

instead of list_memory_data[1] - list_memory_data[0]